### PR TITLE
rebuilt with fixed zoom and narrower maps

### PR DESCRIPTION
**Pull request by dblodgett-usgs**
@jread-usgs pointed out that navigation is a problem on mobile with zoomy maps. I've fixed the zoom on some maps and made all the maps 80% width so you have something other than map to use as a handle to move the page on mobile. Should be good to go!

### CODE REVIEW

File: content/nldi.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 author: David Blodgett
 date: 2017-04-22
 slug: nldi-intro
-draft: True
```
Comment (ldecicco-USGS) [inline content/nldi.md]: Can you put this back in so we can have one last look on dev before pushing it to prod?

Comment (ldecicco-USGS) [inline content/nldi.md]: Actually, no that's fine...we can still make changes before actually clicking the push to prod button.

Comment (ldecicco-USGS) [inline content/nldi.md]: In general though, I'd say that's the last thing to ever do, and should probably be done by the approver.

Comment (dblodgett-usgs) [inline content/nldi.md]: Sorry... didn't think that through completely.

**Pull request merged**